lesson17/script.js

```diff
@@ -0,0 +1,147 @@
+const slider = document.getElementById("js-slider");
+
+const createElement = ({tagName,id,className,text}) => {
+    const element = document.createElement(tagName);
+    if(id){
+        element.id = id;
+    }
+    element.className = className;
+    element.textContent = text;
+    return element;
+}
+
+const createImage = ({id,className,src,alt}) => {
+    const image     = document.createElement("img");
+    if(id){
+        image.id = id;
+    }
+    image.className = className;
+    image.src = src;
+    image.alt = alt;
+    return image;
+}
+
+const createButton = ({id,className}) => {
+    const button = document.createElement("button");
+    button.id = id;
+    button.className = className;
+    return button;
+}
+
+const createErrorMessage = (error) => {
+    const errMsg  = createElement({tagName:"p",className:"error-msg",text:error})
+
+    slider.appendChild(errMsg);
+    return errMsg;
```

Comment:
Q
`slider.appendChild(errMsg);`
Why is it returning when this process is able to output to the screen?
（Where is the returned element used?）

Reply:
Certainly the return value is not used anywhere, so it is unnecessary.
Thank you for teaching.

Reply:
I fixed.
https://github.com/nakagawamai/js-lesson/pull/19/commits/6412b52f4ac65d912459a380a2960ba72ef991c3